layouts/shortcodes/releases.html

```diff
@@ -0,0 +1,3 @@
+{{ $lang := .Page.Lang }}
```

Comment:
You may want to use one of the existing shortcodes provided by the theme:

https://theme.scientific-python.org/shortcodes/

Reply:
Yeah, we could also just use the `include-md` short-code and in the `news.md` for each language, hard code the path to the `releases.md` for that particular language. Figuring out the language in the shortcode and generating the path dynamically might be too clever for its own good. Do you think the first option I mentioned with just `include-md` is better?

Reply:
Perhaps a simpler option is to generate a JSON data file, and to then use a language-specific template to generate the release lines:

https://gohugo.io/templates/data-templates/

Reply:
OK, cool. Thanks @stefanv, I didn't know about this stuff! It looks like I've re-implemented something Hugo is already pretty good at. Data-templates does seem like the way to go.